### ISSUE TITLE
Forum appears to be down

### ISSUE THREAD

**Issue by Tuatarian** [nim-lang/nimforum#267]
https://forum.nim-lang.org/t/2670

Cloudflare gives error 502, bad gateway

**Comment by dom96** [comment]
yeah, looks like this thread belongs to a deleted user and is causing some issues.

**Comment by schneiderfelipe** [comment]
This seems to be down as well: https://forum.nim-lang.org/t/6240
